src/terms.json

```diff
@@ -19,32 +19,53 @@
+        "title": "Transmitera Coronavirus",
+        "aliases": ["Transmitere Coronavirus", "transmitere covid-19", "transmit coronavirus", "transmit covid-19","cum se transmite coronavirus","mod trasmitere coronaviurs","mod de transmitere coronavirusului","transmisilibitatea coronavirusului", "transmisibilitate coronavirus"],
         "explanation": "Coronavirus poate fi transmis de la o persoană la alta, de obicei după un contact strâns cu o persoană infectată. Virusul se transmite mai ales pe cale respiratorie, dar poate intra în corp prin ochi, nas și gură, astfel eviați atingerea dacă nu v-ați spălat bine mâinile. Animalele de companie nu transmit coronavirus.<p>Mai multe informații găsiți aici:</p>",
         "links": [
             "https://www.prefectura.mai.gov.ro/ce-trebuie-sa-stiti-despre-noul-coronavirus/"
         ]
     },
     "stare de urgență": {
         "title": "Stare de urgență începând cu 16 martie",
+        "aliases": ["stare de urgență","stare urgență","starea de urgență","decret stare de urgență","decret stare urgență"],
         "explanation": "<p>Această stare de urgență va face posibilă alocarea de noi resurse importante pentru gestionarea crizei. În acest fel, Guvernul va avea posibilitatea să aloce mai mulți bani domeniului sănătății, mai mulți bani pentru medicamente, mai mulți bani pentru aparatura medicală absolut necesară.</p><p>Mai multe informații găsiți aici:</p>",
         "links": [
-            "https://www.presidency.ro/ro/media/declaratii-de-presa/declaratia-de-presa-sustinuta-de-presedintele-romaniei-domnul-klaus-iohannis1584201636"
+            "https://www.presidency.ro/ro/media/declaratii-de-presa/declaratia-de-presa-sustinuta-de-presedintele-romaniei-domnul-klaus-iohannis1584201636",
+            "https://www.mai.gov.ro/wp-content/uploads/2020/03/Ordonan%C8%9Ba-militar%C4%83-nr.-1-2020-m%C4%83suri-de-prima-urgen%C8%9B%C4%83-Decret.pdf"
         ]
+    },
+    "cazuri":{
+        "title": "Cazuri de Coronavirus",
+        "aliases": ["cazuri","cazuri coronavirus","cazuri de coronavirus","număr infectări coronavirus","număr infecții coronavirus","cazuri covid-19","cazuri de covid-19","număr infectări covid-19","număr infecții covid-19"],
+        "explanation": "<p>Dacă doriți să consultați situația cazulior de cronavirus in timp real, informații găsiți aici:</p>",
+        "links": ["https://datelazi.ro/"]
+    },
+    "diaspora":{
+        "title": "Disapora",
+        "aliases": ["disapora","ce mă fac diaspora","ce mă fac în diaspora","sunt in deplasare","străinătate coronavirus"],
+        "explanation": "<p>Dacă sunteți în străinătate și doriți să aflați informații legate de prevenirea infectării cu virusul COVID-19 (Coronavirus), puteți apela numărul de telefon:+4021.320.20.20</p><p>Mai multe informații găsiți aici:</p>",
+        "links": ["https://diasporahub.ro/","http://www.mae.ro/node/51961"]
```

Comment:
As per [catileptic's comment](https://github.com/code4romania/emergency-news-addon/pull/34#issuecomment-600700952) also remove `diasporahub` for now, please